### ISSUE TITLE
Lots of errors while trying to fine-tune the Speaker diarization pipeline

### ISSUE THREAD

**Issue by Amin-PD** [pyannote/pyannote-audio#534]
```
Using cache found in /root/.cache/torch/hub/pyannote_pyannote-audio_master
Using cache found in /root/.cache/torch/hub/pyannote_pyannote-audio_master
Using cache found in /root/.cache/torch/hub/pyannote_pyannote-audio_master
/usr/local/lib/python3.6/dist-packages/pyannote/audio/embedding/approaches/arcface_loss.py:170: FutureWarning:

The 's' parameter is deprecated in favor of 'scale', and will be removed in a future release

Using cache found in /root/.cache/torch/hub/pyannote_pyannote-audio_master
First trial in progress: : 0trial [00:00, ?trial/s]
Current trial:   0% 0/26 [00:00<?, ?it/s][W 2020-11-27 21:51:25,646] Trial 2 failed because of the following error: KeyError('audio',)
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/optuna/_optimize.py", line 189, in _run_trial
    value = func(trial)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/pipeline/optimizer.py", line 202, in objective
    output = pipeline(input)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speaker_diarization.py", line 154, in __call__
    speech_turns = self.speech_turn_segmentation(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speech_turn_segmentation.py", line 123, in __call__
    sad = self.speech_activity_detection(current_file).get_timeline()
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speech_activity_detection.py", line 139, in __call__
    sad_scores = self._scores(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/wrapper.py", line 280, in __call__
    return self.scorer_(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/base.py", line 149, in __call__
    y, sample_rate = self.raw_audio_(current_file, return_sr=True)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/utils.py", line 237, in __call__
    current_file["audio"], dtype="float32", always_2d=True
  File "/usr/local/lib/python3.6/dist-packages/pyannote/database/protocol/protocol.py", line 137, in __getitem__
    return self._store[key]
KeyError: 'audio'
Traceback (most recent call last):
  File "/usr/local/bin/pyannote-pipeline", line 8, in <module>
    sys.exit(main())
  File "/usr/local/lib/python3.6/dist-packages/pyannote/pipeline/experiment.py", line 443, in main
    pruner=pruner,
  File "/usr/local/lib/python3.6/dist-packages/pyannote/pipeline/experiment.py", line 290, in train
    for i, status in zip(count, iterations):
  File "/usr/local/lib/python3.6/dist-packages/pyannote/pipeline/optimizer.py", line 320, in tune_iter
    self.study_.optimize(objective, n_trials=1, timeout=None, n_jobs=1)
  File "/usr/local/lib/python3.6/dist-packages/optuna/study.py", line 315, in optimize
    show_progress_bar=show_progress_bar,
  File "/usr/local/lib/python3.6/dist-packages/optuna/_optimize.py", line 65, in _optimize
    progress_bar=progress_bar,
  File "/usr/local/lib/python3.6/dist-packages/optuna/_optimize.py", line 156, in _optimize_sequential
    trial = _run_trial(study, func, catch)
  File "/usr/local/lib/python3.6/dist-packages/optuna/_optimize.py", line 189, in _run_trial
    value = func(trial)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/pipeline/optimizer.py", line 202, in objective
    output = pipeline(input)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speaker_diarization.py", line 154, in __call__
    speech_turns = self.speech_turn_segmentation(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speech_turn_segmentation.py", line 123, in __call__
    sad = self.speech_activity_detection(current_file).get_timeline()
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/pipeline/speech_activity_detection.py", line 139, in __call__
    sad_scores = self._scores(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/wrapper.py", line 280, in __call__
    return self.scorer_(current_file)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/base.py", line 149, in __call__
    y, sample_rate = self.raw_audio_(current_file, return_sr=True)
  File "/usr/local/lib/python3.6/dist-packages/pyannote/audio/features/utils.py", line 237, in __call__
    current_file["audio"], dtype="float32", always_2d=True
  File "/usr/local/lib/python3.6/dist-packages/pyannote/database/protocol/protocol.py", line 137, in __getitem__
    return self._store[key]
KeyError: 'audio'

First trial in progress: : 0trial [00:02, ?trial/s]

```
I got this error by running the following code:

`$ pyannote-pipeline train --subset=development --forever ${EXP_DIR} AMI.SpeakerDiarization.MixHeadset`

Please help me to optimize the diarization pipeline using the pre-trained models. However, I want to fine-tune my own model but I got confused!

**Comment by hbredin** [comment]
Solution is available in [this issue](#427). Please search for similar issues before opening a new one.

**Comment by Amin-PD** [comment]
Please double-check your hyperlinks while you are closing an issue.
The link refers to the current issue...

> Solution is available in [this issue](#427). Please search for similar issues before opening a new one.
